src/controllers/busStop.js

```diff
@@ -0,0 +1,98 @@
+import  model  from '../database/models'
+import succesRes from '../helpers/successHandler'
+import errorRes from '../helpers/errorHandler'
+import cryptoRandomString from 'crypto-random-string'
+
+import { Router } from 'express';
+const router = Router();
+
+const { busStop } = model;
+
+
+const createBusStop = async (req,res) => {
+    try{
+        let Stop_id = cryptoRandomString({length: 5, type: 'numeric'})
+        const searchBusStop = await busStop.findAll( { where : req.body } );
```

Comment:
what's the purpose of this line?

Reply:
We used  cryptoRandomString package to generate unique and more detailed IDs for routes same as Bus stops